addons/loyalty/models/loyalty_reward.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models
+
+class LoyaltyReward(models.Model):
+    _name = 'loyalty.reward'
+    _description = 'Loyalty Reward'
+    _rec_name = 'description'
+
+    def _get_discount_mode_select(self):
+        symbol = self.env.context.get('currency_symbol', self.env.company.currency_id.symbol)
```

Comment:
I would document to explain the context thing is because the program_id is not set yet when calling this method.